tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```

Comment:
This is the standard set of bash script options we should use for all scripts, except in special cases. These could be important for helping the person running the script identify when an issue has occurred.

```suggestion
#!/usr/bin/env bash
set -euo pipefail
```